app-misc/ddate/ddate-0.2.2.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit cmake-utils
+
+DESCRIPTION="Perpetual date converter from gregorian to poee calendar"
+HOMEPAGE="https://github.com/bo0ts/ddate"
+SRC_URI="https://github.com/bo0ts/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+LICENSE="public-domain"
+SLOT="0"
+
+RDEPEND="!sys-apps/util-linux[ddate]"
```

Comment:
blocker should not end up in `DEPEND`, easiest to solve by simply define them in reverse.

Reply:
I am not sure, if that's what you want, by I've removed now 'DEPEND'?

Reply:
yes, that's also a way to do it.